NOTES-Windows.txt

```diff
@@ -109,9 +111,8 @@
    "vcvarsall.bat" before you compile. For example, if you want to build
    "arm64" builds, you should run "vcvarsall.bat x86_arm64 uwp".
 
-
```

Comment:
I've wondered, why the removal of the extra empty lines?  They were intentionally there, for more visible separation

Reply:
The ---- is a fairly good seperator..

Reply:
The file util/markdownlint.rb has "multiple blank lines" warning turned on.  So none of the other files have it.

Reply:
So that was a choice made at some point, not having mdl shut up about it.

Reply:
There might have been discussion, but nobody ever "made the choice" because the warning is still in the config file.